src/prted/pmix/pmix_server.c

```diff
@@ -916,22 +919,14 @@ void pmix_server_finalize(void)
     /* finalize our local data server */
     prte_data_server_finalize();
 
-    /* cleanup collectives */
```

Comment:
Something is wrong with this cherry-pick. This change shouldn't be here.

Reply:
Oh I see - https://github.com/openpmix/prrte/pull/1270 was brought over. I don't think it should have, I am reasonably sure that was a master only issue and not an issue on the v2.1 branch. Can you split this out to a revert instead?

Reply:
I can remove it, though I'm puzzled as to why you are saying this PR has a problem? The change you reference came over in an earlier PR and passed CI just fine, and works fine on my local "cluster".

Reply:
@awlauria Looks like this worked fine - can you explain why you think there is something wrong here?

Reply:
I haven't had a chance to test it, but at the time I only intended dc6d6c25fb962c5f3aea9b7dd03578e91 to go back to master, since I think it was the result of switching to PMIx_ utilities. I don't recall seeing the same issue on the v2.1 branch, but will need to verify.

Now here - it seems that part of the above is being reverted, and the git history makes it seem like it was cherry-picked from 58ed3dd5c29768a90f8af153aa20205885e6c62c.

Could you separate these out into a separate commit with a note on why these changes were reverted? Right now it looks like it went in on a rebase/squash on accident.

I'll test tomorrow to see if it should be brought back. It's probably fine, but we are looking to update the submodule pointer later this week. So would like to be safer than sorry.

Reply:
Okay, I'll take a closer look at how this came together. There are conflicts that need to be resolved, so it is possible that something got caught there.